pip/pep425tags.py

```diff
@@ -32,8 +32,18 @@ def get_impl_ver():
 def get_platform():
     """Return our platform name 'win32', 'linux_x86_64'"""
     # XXX remove distutils dependency
-    return distutils.util.get_platform().replace('.', '_').replace('-', '_')
-
+    platform = distutils.util.get_platform().replace('.', '_').replace('-', '_')
+
+    if platform == 'linux_x86_64':
+        # HACK: work around get_platform() claiming we're running in 64-bit
+        #       mode just because we have a 64-bit kernel, even though this
+        #       Python version has been compiled for 32-bit mode.
+        import struct
+        ptr_size = struct.calcsize('P') * 8
+        if ptr_size == 32:
+            return 'linux_x86_32'
```

Comment:
@ogrisel that could very well be, but frankly I don't even recall where I encountered this problem. It's been two years and I don't have a system with such a setup anymore to test this on. All I know is that this fix/workaround worked for me at the time.